src/python/aqueduct_executor/operators/connectors/tabular/connector.py

```diff
@@ -31,7 +35,12 @@ def extract(self, params: extract.Params) -> pd.DataFrame:
         """
 
     @abstractmethod
-    def load(self, params: load.Params, df: pd.DataFrame) -> None:
+    def load(
+        self,
+        # TODO: https://linear.app/aqueducthq/issue/ENG-1285/revisit-python-typings-for-connector-code
```

Comment:
nit: can we format these like `TODO(ENG-1285): {short human readable description}`